zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnJobBehavior.java

```diff
@@ -7,26 +7,32 @@
  */
 package io.camunda.zeebe.engine.processing.bpmn.behavior;
 
+import static io.camunda.zeebe.util.Either.right;
+import static java.util.Optional.of;
+import static java.util.Optional.ofNullable;
```

Comment:
🔧 Static members should be accessed via their class according to our code style

https://google.github.io/styleguide/javaguide.html#s6.3-static-members

Reply:
👍 Updated the code to refer to static methods using their class name.